### PR TITLE
EJB bindToServerRootConfig element tests

### PR DESCRIPTION
**Pull request by olendvcook**
for #8757 but  #13338 will be when it's actually done.

### CODE REVIEW

**Comment by olendvcook** [comment]
#build

File: dev/com.ibm.ws.ejbcontainer.bindings_fat/fat/src/com/ibm/ws/ejbcontainer/bindings/fat/tests/BindToServerRootTest.java [hunk 1]
```diff
@@ -0,0 +1,129 @@
+/*******************************************************************************n * Copyright (c) 2020 IBM Corporation and others.n * All rights reserved. This program and the accompanying materialsn * are made available under the terms of the Eclipse Public License v1.0n * which accompanies this distribution, and is available atn * http://www.eclipse.org/legal/epl-v10.htmln *n * Contributors:n *     IBM Corporation - initial API and implementationn *******************************************************************************/
```
Comment (tkburroughs) [inline dev/com.ibm.ws.ejbcontainer.bindings_fat/fat/src/com/ibm/ws/ejbcontainer/bindings/fat/tests/BindToServerRootTest.java:1]: Copyright is missing returns

File: dev/com.ibm.ws.ejbcontainer.bindings_fat/test-applications/ConfigTestsWeb.war/src/com/ibm/ws/ejbcontainer/bindings/configtests/web/BindToServerRootServlet.java [hunk 2]
```diff
@@ -0,0 +1,154 @@
+/*******************************************************************************n * Copyright (c) 2020 IBM Corporation and others.n * All rights reserved. This program and the accompanying materialsn * are made available under the terms of the Eclipse Public License v1.0n * which accompanies this distribution, and is available atn * http://www.eclipse.org/legal/epl-v10.htmln *n * Contributors:n *     IBM Corporation - initial API and implementationn *******************************************************************************/
```
Comment (tkburroughs) [inline dev/com.ibm.ws.ejbcontainer.bindings_fat/test-applications/ConfigTestsWeb.war/src/com/ibm/ws/ejbcontainer/bindings/configtests/web/BindToServerRootServlet.java:1]: Copyright is missing returns

Comment (olendvcook) [reply dev/com.ibm.ws.ejbcontainer.bindings_fat/test-applications/ConfigTestsWeb.war/src/com/ibm/ws/ejbcontainer/bindings/configtests/web/BindToServerRootServlet.java:1]: I forgot my eclipse does this for new files oops, I'll fix it for the next PR

**Comment by tkburroughs** [comment]
@olendvcook merged

**Comment by tkburroughs** [comment]
1 failing FAT is unrelated to changes; occurs in other builds

**Pull request merged**